internal/controller/components/modelregistry/modelregistry_support.go

```diff
@@ -28,7 +28,6 @@ var (
 		"IMAGES_MODELREGISTRY_OPERATOR": "RELATED_IMAGE_ODH_MODEL_REGISTRY_OPERATOR_IMAGE",
 		"IMAGES_GRPC_SERVICE":           "RELATED_IMAGE_ODH_MLMD_GRPC_SERVER_IMAGE",
```

Comment:
```suggestion
```
@zdtsw Can we remove this also? We are already no longer using the image this will point to, and I'm cleaning up references to this in https://github.com/opendatahub-io/model-registry-operator/pull/348

I want to make sure it won't break anything when I merge that PR, per [this comment](https://github.com/opendatahub-io/model-registry-operator/pull/348#discussion_r2467915467)

Reply:
we have another [PR](https://github.com/opendatahub-io/opendatahub-operator/pull/2349) to get rid of `RELATED_IMAGE_ODH_MLMD_GRPC_SERVER_IMAGE` and we will need devops to remove reference in build as well